bugz.py

```diff
@@ -146,6 +152,13 @@ def __init__(self):
         super().__init__()
         self.db = Database()
 
+    def qa_needed_delete(self):
```

Comment:
if I remember correctly, this is be cause we're using the DB as temp cache for this data so wiping it before rehydrating it is the way to go.  true?

Reply:
That's the case for TestRail test suites. In this case I want to clean the table so that it shows daily the active bugs that require qa attention